routes/web.php

```diff
@@ -13,30 +13,18 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get('/', 'App\Http\Controllers\HomeController@index');
+
+Route::get('/user/{id}', 'App\Http\Controllers\HomeController@show')->name('user');
```

Comment:
Ovo je ok ali ponavljamo se, hajde da istrazimo grupisanje ruta, gde cemo proslediti putanju do kontrolera. Istraziti malo https://stackoverflow.com/questions/43313989/laravel-5-5-passing-controllers-to-routes-groups